Ghidra/Processors/PowerPC/data/languages/ppc_instructions.sinc

```diff
@@ -17,16 +17,20 @@
 
 #addo r1,r2,r3  0x7c 22 1e 14
 :addo D,A,B		is OP=31 & D & A & B & OE=1 & XOP_1_9=266 & Rc=0
-{ 
+{
+	local tmpA = A;
```

Comment:
For most of these, you could avoid the extraneous assignment to locals by swapping the order of the assignment to destination and the calculation of the flags. There are some cases where that is not suitable - such as `fadd`

Reply:
doh.  thanks, not sure why that didn't click.  I didn't see a quick/easy way to avoid this in floating point, but all the non-fp could be re-ordered.